### ISSUE TITLE
improve UX for display of template list on "Create New"

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#1841]
@astridu there's a new feature in Sinopia where, for fields that are lookups to Sinopia Works or Sinopia Instances, we display a "Create New" menu with a list of templates whose resourceURI = bf:Work or bf:Instance, as appropriate.

currently, we only display the first 25 of the matching templates (in alpha order by label), but users need to see all the matching templates to find the one they want (or have some other way to find the one they want)

can you suggest a better design to show a longer list of matching templates? or to have some way to filter the list of matching ones to find what they want? it's unclear what the upper range is as far as # of templates that might match

one use case is a user from Yale was expecting to see a Yale template listed and it wasn't because it was lower in the alphabet, so in that case they knew they wanted one whose label contained Yale, so if there was some kind of filter/type-ahead they could use that

here's an example template
https://stage.sinopia.io/editor/sinopia:work:instance:lookup

click Add next to `this is a lookup of Sinopia instances` and same with Works, and you'll see the menu

(ignore the third field (lookup to all Sinopia stuff) because it's not supposed to have a "create new" menu, that's a bug)

**Comment by astridu** [comment]
I think the search box is a good start. Some other options:
1. Filter by (same UI pattern as the other filter by for record search)
![image](https://user-images.githubusercontent.com/42216982/69452843-8bdac280-0d17-11ea-9c5c-cd5a994cb673.png)

2. Navigate by alphabet
![image](https://user-images.githubusercontent.com/42216982/69452868-9bf2a200-0d17-11ea-9cbb-b4732bb737ba.png)

**Comment by justinlittman** [comment]
@astridu Could we just do a plain-old dropdown? That would take up less screen real estate and supports basic selection.

**Comment by astridu** [comment]
@justinlittman A dropdown that contains all the templates? How many templates could that be?

**Comment by jgreben** [comment]
In planning meeting we decided to use a plain 'ol drodown.

**Comment by michelleif** [comment]
@justinlittman in Create New menu, it still looks like we are showing only the first 25 matching templates, instead of **all** matching templates

In DEV, I uploaded a new template (label: `ZZ most random instance`, id: `zz:random:Instance`) with class bf:Instance, expecting it to show up in list that is presenting for Sinopia Instance lookups. It doesn't show up, presumably because there are already 25 items in the list.

Then I uploaded a new template called `aa most random instance` and went to a Sinopia instance lookup field again and the aa... template *does* show up, presumably because it files high in the list alphabetically.

This is the template I'm using to see the actual lookup field and the Create New menu:
https://development.sinopia.io/editor/sinopia:work:instance:lookup
